gdcdictionary/schemas/serious_adverse_event.yaml

```diff
@@ -0,0 +1,192 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "serious_adverse_event"
+title: Serious Adverse Event
+type: object
+namespace: https://jcoin.datacommons.io/dd
+category: administrative
+program: '*'
+project: '*'
+description: >
+  The most up to date information pertaining to a single SAE. 
+  For example, if an SAE is submitted and the a follow up form is submitted with more up to date
+  information, this node will capture the most up to date information. However,
+  all files (ie initial and follow ups will be stored in a child node called serious_adverse_event_files)
+
+  Generated through the serious adverse event workflow node
+additionalProperties: false
+submittable: true
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+  - state
+
+#only need up to date info (so one link to protocol) del the files/workflows
```

Comment:
there are several commented out blocks in this file, should we clean them up or are you keeping them for future use?

Reply:
keeping them in for (potential) future use/reference